libiocage/lib/Jail.py

```diff
@@ -60,19 +62,24 @@ class Jail:
             release (stored in `zpool/iocage/base/<RELEASE>)
     """
 
-    def __init__(self, data={}, zfs=None, host=None, logger=None, new=False):
+    def __init__(self,
+                 data: Union[str, dict]={},
+                 zfs: Optional[libzfs.ZFS]=None,
+                 host: Optional[libiocage.lib.Host]=None,
+                 logger: Optional[libiocage.lib.Logger]=None,
```

Comment:
for these two, i'm getting

```
libiocage/lib/Jail.py:68: error: Invalid type "libiocage.lib.Host"
libiocage/lib/Jail.py:69: error: Invalid type "libiocage.lib.Logger"
```

Reply:
new data! this is a module, not a type

how do i access the actual type, or the class?

Reply:
libiocage.lib.Host.Host 🌮

Reply:
but that's uglt 😅

Reply:
Yeah, I know! The other option is to put everything in one file. Let's quickly forget that.

Yet another solution is to mess with `sys.path`, but @william-gr taught me this is a no-go for libraries and bad practice everywhere else.

I think `lib/__init__.py` is an appropriate solution: The public exposed classes are available as `libiocage.Jail`, `libiocage.Release`, etc when importing the directory (for development) or importing globally installed libiocage.